StructFields.pm

```diff
@@ -92,8 +94,8 @@ our $cur_init_value = undef;
 sub add_simple_init($);
 
 my %custom_primitive_handlers = (
-    'stl-string' => sub { return "std::string"; },
-    'stl-fstream' => sub { return "std::fstream"; },
+    'stl-string' => sub { header_ref("string"); return "std::string"; },
```

Comment:
These are fine with me, although like above, maybe it would be easier to just include these STL headers in one centralized file.